src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -465,6 +465,10 @@ public static ListBoxModel fillAuthenticationItems(Item project, String url) {
         public static List<Range<Integer>> parseToRange(String value) {
             List<Range<Integer>> validRanges = new ArrayList<Range<Integer>>();
 
+            if (Strings.isNullOrEmpty(value)) {
```

Comment:
I think, to avoid compatibility break, we could parse the default value, WDYT?

```
 if (Strings.isNullOrEmpty(value)) {
  value = HttpRequest.DescriptorImpl.validResponseCodes
}
//... and keep the parse until return it
```